.github/workflows/publish-docker.yaml

```diff
@@ -73,10 +74,10 @@ jobs:
         uses: docker/setup-qemu-action@v1
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
-      - name: Build and push docker images
-        run: make build.all docker.push
       - name: Build and push docker images based on Java 17
         env:
           SW_OAP_BASE_IMAGE: eclipse-temurin:17-jre
           TAG: ${{ env.TAG }}-java17
         run: make build.all docker.push
+      - name: Build and push docker images
+        run: make build.all docker.push
```

Comment:
I swap this so the `latest` tag will remain for `java11` and `9.4.0-java-17` for java 17